universal-login-relayer/test/integration/integration/sql/MessageRepository.ts

```diff
@@ -34,7 +35,7 @@ describe(`INT: IMessageRepository (${config.type.name})`, async () => {
       args = knex;
     }
     messageRepository = new config.type(args);
-    message = createSignedMessage({from: walletContract.address, to: '0x'}, wallet.privateKey);
+    message = createSignedMessage({from: walletContract.address, to: '0x', nonce: await walletContract.lastNonce()}, wallet.privateKey);
```

Comment:
Why do we need to add nonce all of a sudden?

Reply:
Because lately nonce was hard-coded as 0 and we setup `walletContract` manually (`walletContract.addKey(...)`, `walletContract.setRequiredSign(..)`) but now we use `executeSigned` to setup contract and nonce is no more 0, it is changing.